agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/TelemetryClientInitializer.java

```diff
@@ -95,6 +98,11 @@ public static void initialize(TelemetryClient telemetryClient, Configuration con
     setQuickPulse(configuration, telemetryClient);
   }
 
+  private static boolean isAgentRunningInSandboxEnvWindows() {
+    String qualifiedSdkVersion = PropertyHelper.getQualifiedSdkVersionString();
+    return qualifiedSdkVersion.startsWith("awr") || qualifiedSdkVersion.startsWith("fwr");
```

Comment:
can we (easily) cover cases where users are running the standalone agent in App Services?